seclea_ai/seclea_ai.py

```diff
@@ -904,15 +903,15 @@ def _get_framework(model) -> ModelManagers:
         else:
             return ModelManagers.NOT_IMPORTED
 
-    def _update_dataset_metadata(self, dataset_hash, metadata):
+    def _update_dataset_metadata(self, dset_pk, metadata):
         """
         Update the dataset's metadata. For use when the metadata is too large to encode in the url.
-        @param dataset_hash:
+        @param dset_pk:
         @param metadata:
         @return:
         """
         res = self._transmission.patch(
-            url_path=f"/collection/datasets/{dataset_hash}",
+            url_path=f"/collection/datasets/{dset_pk}",
             obj={
                 "metadata": metadata,
             },
```

Comment:
This function needs to be removed and the usage updated to reflect that. This was to get around size limit in query params which is removed in this PR.